### ISSUE TITLE
Alert Group has axe violations

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-org#1935]
Currently the alert group has axe violations: id attribute value must be unique (issue with overview heading)

(There is currently a link violation as well, but it should be resolved by https://github.com/patternfly/patternfly-org/pull/1588)

**Comment by redallen** [comment]
Closing in favor of #2158.
